docs/components/console/manage-organization/external-sso.md

```diff
@@ -0,0 +1,10 @@
+---
+id: external-sso
+title: Connect your IDP with Camunda Platform
+---
+
+## Connecting your Identity Provider with Camunda Platform
```

Comment:
@ultraschuppi should we include the Keyword "SSO" in the title?

Reply:
good point! done in https://github.com/camunda/camunda-platform-docs/pull/1338/commits/f6c482e6a5b40a4cdf47008feb9a1200fcaeb3b6